### DISCUSSION TITLE
"the subscription is not allowed to create or update the serverfarm." - Erro na criação de Web App

### DISCUSSION THREAD

**Discussion by EduardoKroetz** [balta-io/forum#542] [Tira Dúvidas] [answered]
Quando fui criar um Web App não deu certo, ficou ocorrendo esse erro:
"the subscription <subscriptionId> is not allowed to create or update the serverfarm.". Testei criar um Sql Database e funcionou, porém não consigo criar um web app. 
OBS: Criei e recriei isso umas 10 vezes.
![Screenshot_4](https://github.com/balta-io/forum/assets/146035630/411422e6-fc81-4a03-b6fa-0848ba46d19b)
![image](https://github.com/balta-io/forum/assets/146035630/4fa3b6ad-d9e1-4d06-a72e-708701fffedb)
Erro completo:
{
    "status": "Failed",
    "error": {
        "code": "Forbidden",
        "message": "The subscription <minhaSubscrição> is not allowed to create or update the serverfarm.",
        "details": [
            {
                "message": "The subscription <minhaSubscrição> is not allowed to create or update the serverfarm."
            },
            {
                "code": "Forbidden"
            },
            {}
        ]
    }
}

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @EduardoKroetz, como vai?

Acredito que sua conta não tenha acesso para criação deste recurso. Você pode abrir um chamado no Azure para eles verificarem esta permissão e liberar o recurso.
